staff/jordi-pujol/gamma-template/contact.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+        crossorigin="anonymous">
+    <title>Contact</title>
+</head>
+
+<body class="contact">
+    <header class="cheader">
+        <nav class="cheader__menu">
+            <p>Gamma</p>
+            <p class="cheader__menu--display"><a href="home.html">Home</a></p>
+            <p class="cheader__menu--display"><a href="about.html">About</a></p>
+            <p class="cheader__menu--display">Contact</p>
+            <i class="fas fa-bars" id="burger"></i>
+        </nav>
+        <h1 class="cheader__title">Contact Us</h1>
+    </header>
+    <main class="cmain">
+        <section class="cmain__form">
+            <p class="cmain__form--item">Name</p>
+            <form action=""></form> <input class="cmain__form--text" type="text" placeholder="Enter name">
```

Comment:
a single tag From has to include all the entries and the button

Reply:
Thanks! I've just changed it